README.md

```diff
@@ -33,6 +33,7 @@
 
 - [use Rspack](https://rspack.dev/guide/quick-start.html)
 - [use Rsbuild](https://rsbuild.dev/guide/start/quick-start.html)
+- [electron-forge-rspack-template](https://github.com/noshower/electron-forge-rspack-template)
```

Comment:
Can you move this template to the `Starter` section? The "Starter" is for templates for scanfolds.

Reply:
moved